docker/Dockerfile.ubuntu.xenial

```diff
@@ -22,6 +22,7 @@ RUN apt-get update && apt-get -y install --no-install-suggests --no-install-reco
   gcc \
   git \
   jq \
+  junit \
```

Comment:
I think you should only need to add `junit` for the bionic image. I don't think it'll be used by the xenial or Debian testing images.

Reply:
Actually, do we need junit at all on the images?

Reply:
Yes, it's a long shot... seems not needed, tried it anyway.